VotingApplication/VotingApplication.Web/App_Start/BundleConfig.cs

```diff
@@ -93,6 +93,15 @@ public static void RegisterBundles(BundleCollection bundles)
             angularRoute.Include("~/Scripts/Lib/angular-route-min.js");
             bundles.Add(angularRoute);
 
+            var angularMessages = new Bundle("~/Bundles/ScriptLib/AngularMessages", string.Format("{0}/angular-messages.min.js", angularCdnBase));
+            angularMessages.Include("~/Scripts/Lib/angular-messages-min.js");
+            bundles.Add(angularMessages);
+
+            // See: http://plnkr.co/edit/gBDV2ABghTMqqs5YaElB?p=preview
+            // From: https://github.com/angular/material/issues/1269
+            var angularThemeColors = new Bundle("~/Bundles/ScriptLib/AngularThemeColors");
```

Comment:
Bundling has generally been done with dependencies appearing before things that use them. AngularThemeColors requires Angular Material, so should be below it.